shared/enclave/preload.ts

```diff
@@ -4,6 +4,6 @@ import { PROTOCOL_NAME } from 'shared/enclave/utils';
 export function registerProtocol() {
   // Whitelist custom protocol
   protocol.registerSchemesAsPrivileged([
-    { scheme: PROTOCOL_NAME, privileges: { standard: true, secure: true } }
+    { scheme: PROTOCOL_NAME, privileges: { standard: true, secure: true, bypassCSP: true } }
```

Comment:
This is way too permissive even in electron: https://www.contextis.com/us/blog/basic-electron-framework-exploitation
How about we add the required trezor domain to our CSP instead?

Reply:
This is exclusively for the protocol that accesses the enclave. Is it still an issue then?

Reply:
Wouldn't it be possible to add the protocol itself to CSP? e.g. `connect-src eth-enclave:;`?

Reply:
Maybe - but we only want it to be set for Electron.